torch/testing/_internal/common_methods_invocations.py

```diff
@@ -39,7 +39,7 @@
     TEST_WITH_ROCM, IS_FBCODE, IS_WINDOWS, IS_MACOS, IS_S390X, TEST_SCIPY,
     torch_to_numpy_dtype_dict, numpy_to_torch_dtype, TEST_WITH_ASAN,
     GRADCHECK_NONDET_TOL, slowTest, TEST_WITH_SLOW,
-    TEST_WITH_TORCHINDUCTOR, MACOS_VERSION
+    TEST_WITH_TORCHINDUCTOR, MACOS_VERSION, skipIfXpu,
```

Comment:
```suggestion
    TEST_WITH_TORCHINDUCTOR, MACOS_VERSION, skipIfXpu
```

Reply:
@EikanWang  seems a lot of imports in this file with ',' added after the last one, [link1](https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_methods_invocations.py#L35) and [link2](https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_methods_invocations.py#L25)